project-templates/java-emf-theia/glsp-client/README.md

```diff
@@ -0,0 +1,72 @@
+# Eclipse GLSP - Project Template:<br> 🖥️ Java ● 🗂️ EMF ● 🖼️ Theia
+
+## GLSP Client
+
+This directory provides the initial setup of the package architecture and environment for a GLSP Client that is integrated into a Theia browser application.
+It is based on the `TaskList` example diagram language.
+
+For more detailed instructions and information please checkout the [general README](../README.md) in the parent directory.
+
+## Building
+
+To build all GLSP client packages simply execute the following in the `glsp-client` directory:
+
+```bash
+yarn
+```
+
+## Running the example
+
+To start the Theia browser application with the integrated `TaskList` example execute:
+
+```bash
+yarn start
+```
+
+It's also possible to start the Theia browser application in external mode. This means
+the `TaskList` GLSP server will not be started as an embedded process and is expected to be already running. This can be used for debugging purposes:
```

Comment:
```suggestion
It's also possible to start the Theia browser application in external mode.
This means the `TaskList` GLSP server will not be started as an embedded process and is expected to be already running.
This can be used for debugging purposes, where you first start the GLSP server in debug mode and let the Theia application connect to it:
```